libr/core/cmd_help.inc.c

```diff
@@ -471,7 +474,10 @@ static char *filterFlags(RCore *core, const char *msg) {
 			if (!end) {
 				end = dollar + strlen (dollar);
 			}
-			word = r_str_newlen (dollar+1, end-dollar-1);
+			word = NULL;
+			if (end-dollar-1) {
```

Comment:
```suggestion
			if (end - dollar - 1 > 0) {
```

dont assign word twice, use else branch